charts/velero/values.yaml

```diff
@@ -284,8 +284,12 @@ configuration:
   # here if using a non-default value. The `velero server` default values are shown in the
   # comments below.
   # --------------------
+  # `velero server` default: restic
```

Comment:
You don't have to fix this in this PR, but if these values are to be set by the user, we better add more explanations in the comment